src/app/home/panel/cards/transactions/transactions-list/transactions-list.component.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../../../../../assets/styles/colors";
+
+:host {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.transaction-info-container {
+  width: 100%;
+  display: flex;
+  padding: 1rem 0;
+  border-bottom: $light-gray 1px solid;
+  margin: .5rem 0;
+}
+
+.info-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 .5rem;
+}
+
+.info-name {
+  font-weight: bold;
+}
+
+.info-date {
+  font-size: .8rem;
+  font-weight: 300;
+}
+
+.info-amount {
+  flex: 1;
+  align-items: flex-end;
+  font-weight: bold;
+}
```

Comment:
Try to encapsulate styles for naming noise reduction.

Reply:
Ok